docs/reference/commandline/daemon.md

```diff
@@ -643,4 +644,20 @@ set like this:
     /usr/local/bin/docker daemon -D -g /var/lib/docker -H unix:// > /var/lib/docker-machine/docker.log 2>&1
 
 
+# Default cgroup parent
```

Comment:
@moxiegirl:  What you have is a good start. Couple of high level comments:
1. This option is available both for "docker daemon" and "docker run". We can add help at the daemon level and reference it from the run subcommand (or vice versa).
2. Given that the behaviour is different depending on native.cgroupdriver choice, it's better to split this section by the same (native.cgroupdriver)

Proceeding to review what you have for cgroupfs:
- Avoid having references to the cgroup memory subsystem alone. For ex, /sys/fs/cgroup/memory. All cgroup subsystems listed in /proc/subsystems will have the newly created cgroup.
- Reference to daemoncgroup sounds incorrect. Can you point the source of this info? 

For cgroup-parent option on systemd cgroups, you can use the following content:

When cgroups are managed by systemd (using --exec-opt native.cgroupdriver=systemd), containers are created by default under system.slice. Custom systemd slices can be used to group containers and control resource limits on them. Custom slices can be created by adding slice units in /etc/systemd/system. For example, create a limits.slice (ref. http://www.freedesktop.org/software/systemd/man/systemd.resource-control.html)

Run containers in limits.slice using --cgroup-parent:
docker run -it --cgroup-parent=limits.slice busybox sh
docker run -it --cgroup-parent=limits.slice ubuntu bash

Using systemd-cgls, observe that both containers are created under limits.slice. Note that limits.slice is in the same hierarchical level as system.slice.

├─system.slice
│ ├─thermald.service
│ │ └─654 /usr/sbin/thermald --no-daemon --dbus-enable
│ ├─dbus.service
│ │ ├─  728
...

├─limits.slice
│ ├─docker-d8119d36eea49874ddfa8e8f83c3aa72730b3e57ae6cb43bfd4d299154a24a5c.scope
│ │ └─2330 bash
│ └─docker-d4f053a9bbefb30831ead236ed9716522f3eb2cc50d3443f30ea41f4da766939.scope
│   └─2284 sh

For each container, a docker-<containerID>.scope is created under /sys/fs/cgroup/<subsystem>/<cgroup-parent